src/OracleService/OracleService.csproj

```
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo.ConsoleService" Version="1.0.0" />
+    <PackageReference Include="Neo.ConsoleService" Version="1.2.0" />
     <PackageReference Include="NeoFS.API" Version="3.0.3" />
```

```suggestion
    <PackageReference Include="NeoFS.API" Version="3.1.0" />
```

It has been updated to `v3.1.0`.